.evergreen/config.yml

```diff
@@ -442,6 +443,33 @@ functions:
           OCSP_TLS_SHOULD_SUCCEED="${OCSP_TLS_SHOULD_SUCCEED}" \
           make evg-test-ocsp
 
+  run-versioned-api-test:
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src/go.mongodb.org/mongo-driver
+        script: |
+          ${PREPARE_SHELL}
+          export GOFLAGS=-mod=vendor
+          set +o xtrace
+          AUTH="auth" \
```

Comment:
Passing `AUTH=auth` and `SSL=nossl` directly here is fine because we know those will be the correct values, but that might not necessarily be true in the future if we add more variants for newer API versions. Can we instead have `test-standalone-versioned-api` pass up AUTH/SSL variables that we then reference here? See the existing [test-standalone-auth-nossl](https://github.com/mongodb/mongo-go-driver/blob/60f76f5b1810553090503950fce37c876253217b/.evergreen/config.yml#L891) for an example.